src/Provider/Mapzen/Tests/MapzenTest.php

```diff
@@ -99,9 +99,9 @@ public function testReverseWithRealCoordinates()
         $this->assertEquals('Gage Street', $result->getStreetName());
         $this->assertEquals('LA1 1UH', $result->getPostalCode());
         $this->assertEquals('Lancaster', $result->getLocality());
-        $this->assertCount(4, $result->getAdminLevels());
+        $this->assertNotEmpty($result->getAdminLevels());
         $this->assertEquals('Lancashire', $result->getAdminLevels()->get(1)->getName());
-        $this->assertEquals('England', $result->getAdminLevels()->get(3)->getName());
+        $this->assertEquals('England', $result->getAdminLevels()->get(4)->getName());
```

Comment:
What if you change this back to 3?

Inspect and see what level that would give you "England".

Reply:
How do I run a test on my computer to see what levels are returned?

Reply:
You can run only Mapzen by
```
cd src/Provider/Mapzen
composer update
./vendor/bin/phpunit
```